package.json

```diff
@@ -12,16 +12,15 @@
     "@lblod/mu-auth-sudo": "^0.6.1",
     "jsonld": "^8.3.1",
     "JSONStream": "^1.3.5",
-    "lodash.flatten": "^4.4.0",
-    "lodash.uniq": "^4.5.0",
-    "lodash.uniqwith": "^4.5.0",
+    "lodash": "^4.17.21",
     "n3": "^1.17.0",
     "rdf-serialize": "^2.2.2",
     "rdfa-streaming-parser": "^2.0.1",
     "stream-to-array": "^2.3.0",
     "stream-to-string": "^1.2.0",
     "streamify-array": "^1.0.1",
-    "streamify-string": "^1.0.1"
+    "streamify-string": "^1.0.1",
+    "uuid": "^9.0.0"
```

Comment:
It seems the mu-javascript template only exposes v1. Hence the explicit re-addition.